main.py

```diff
@@ -172,6 +176,12 @@ def percent(part, whole):
 		print("Error loading blocker configuration data: ", e)
 		sys.exit()
 
+    # Get set from the list of all bugs in all jobs
+	all_bugs_set = get_bugs_set(blockers)
+
+	# Create dictionary the set of all bugs (key) with name and link as value
+	all_bugs_dict = get_bugzilla(list(all_bugs_set))
```

Comment:
```suggestion
	all_bugs_dict = get_bugzilla(all_bugs_set)
```
We can removed the `list()` cast; I tested without it and it works